portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +23,30 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private ArrayList<String> comment = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Jennifer!</h1>");
+    //response.setContentType("text/html;");
+    //response.getWriter().println("Hi! I'm Jennifer,");
+    //response.setContentType("application/json;");
+    //String json = convertJson(response);
+    //response.getWriter().println(json);
+  }
+     @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    comment.add(request.getParameter("username"));
+    comment.add(request.getParameter("title"));
+    comment.add(request.getParameter("message"));
+    response.sendRedirect("/comments.html");
+
```

Comment:
Formatting: no new line at the end of functions, add a new line in between functions.